packages/generator-converters/src/wirePropertyV2ToSdkPropertyDefinition.ts

```diff
@@ -102,10 +108,23 @@ function objectPropertyTypeToSdkPropertyDefinition(
       } else if (propertyType.itemType?.type === "double") {
         return "numericTimeseries";
       } else return "sensorTimeseries";
+    case "struct": {
+      return propertyType.structFieldTypes.reduce(
+        (structMap: Record<string, SimpleWirePropertyTypes>, structField) => {
+          structMap[structField.apiName] =
+            objectPropertyTypeToSdkPropertyDefinition(
+              structField.dataType,
+            ) as SimpleWirePropertyTypes;
```

Comment:
This is fine because right now, structs can only be field -> simple property type, cannot have nested structs yet in OSS